payment_gateway_paypal/services/payment_service.py

```diff
@@ -3,9 +3,10 @@
 # @author Sébastien BEAU <sebastien.beau@akretion.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import models
+from odoo.exceptions import Warning as UserError
+from odoo.addons.component.core import Component
+
 import json
-from openerp.exceptions import Warning as UserError
 import logging
 _logger = logging.getLogger(__name__)
 
```

Comment:
We should migrate and refactor the profile creation, (line 20) method "def create_profile".
We should create a wizard that will give the posibility ro configure the profile